src/openforms/submissions/templates/submission_logs/submission_logs.html

```diff
@@ -0,0 +1,69 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static %}
+
+{% block extrastyle %}
+
+<link rel="stylesheet" href="{% static "bundles/core-css.css" %}">
+{{ block.super }}
+{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; <a href="{% url 'admin:submissions_submission_change' original.pk %}">{{ original }}</a>
+&rsaquo;{% trans 'Evaluated Logic Logs'|capfirst%}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans  "Logging" %}</h1>
+    {% block field_sets %}
+        {% for log in logs_activity  %}
+            <fieldset class="module aligned collapse collapsed">
+                <h2>{{ log.name|date:'Y-m-d H:i:s' }} - {% trans 'Logic' %} - </h2>
```

Comment:
is `.name` a date/datetime?

Reply:
It is a date, small confusion because it is used as the title of the log entry, I change that name